junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileArgumentsProvider.java

```diff
@@ -105,7 +105,7 @@ public boolean hasNext() {
 
 		@Override
 		public Arguments next() {
-			Arguments result = arguments(this.nextCsvRecord);
+			Arguments result = of(this.nextCsvRecord);
```

Comment:
Fine. As long **javac** and **spotless** are happy...

Hey! What about calling the method `$`? ;)